.github/workflows/build_test.yml

```diff
@@ -17,40 +17,56 @@ on:
 jobs:
   ubuntu_build:
     name: Ubuntu Build ${{ matrix.name }}
-    runs-on: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
```

Comment:
`${{ matrix.os || 'ubuntu-latest' }}` and you can remove all the `os: ubuntu-latest` lines.

Reply:
Done. Thanks